bedrock/base/redirects.py

```diff
@@ -0,0 +1,7 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from bedrock.redirects.util import redirect
+
+redirectpatterns = (redirect(r"^vpn/download/?$", "products.vpn.download"),)
```

Comment:
Suggestion: add a comment here to explain why this redirect is here, and not in `products/redirects.py`?